.github/workflows/cmake.yml

```
@@ -433,177 +425,184 @@ jobs:
+            # Codecov disabled for clang, errors
 
           # Check for undefined symbols in the version script for the modern api
-          - name: Ubuntu Clang Undefined Symbols
+          - name: Ubuntu Clang-20 Undefined Symbols
             os: ubuntu-latest
-            compiler: clang-15
-            cxx-compiler: clang++-15
+            compiler: clang-20
+            cxx-compiler: clang++-20
             cmake-args: -DCMAKE_SHARED_LINKER_FLAGS="-fuse-ld=lld -Wl,--no-undefined-version" -DZLIBNG_ENABLE_TESTS=OFF
             build-shared: ON
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)